<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ContentManagementSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDiscard">
         <property name="text">
          <string>Discard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchLine">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAddItem">
         <property name="text">
          <string>Add Item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDeleteItem">
         <property name="text">
          <string>Delete Item</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>Context</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Content</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/new.png</normaloff>:/ics/ics/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/open.png</normaloff>:/ics/ics/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/save.png</normaloff>:/ics/ics/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/export.png</normaloff>:/ics/ics/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/exit.png</normaloff>:/ics/ics/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/copy.png</normaloff>:/ics/ics/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/paste.png</normaloff>:/ics/ics/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/cut.png</normaloff>:/ics/ics/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/undo.png</normaloff>:/ics/ics/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ics/ics/redo.png</normaloff>:/ics/ics/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
